cmd/run.go

```diff
@@ -56,6 +57,13 @@ func RunCmd(lg log.Logger, mt *metrics.StatsdMonitor, cfg config.Config) *cobra.
 			"group:core": "true",
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if configFile != "" {
+				var err error
+				cfg, err = config.LoadFromPath(configFile)
+				if err != nil {
+					return err
```

Comment:
why not just

```suggestion
				cfg, err := config.LoadFromPath(configFile)
				if err != nil {
					return err
```
?

Reply:
Wouldn’t that make `cfg` a local variable as well??